### ISSUE TITLE
public-nlb-healthcheck-sources support and public-sos-endpoints in exoscale_security_group_rule ressource

### ISSUE THREAD

**Issue by fegger-ducksify** [exoscale/terraform-provider-exoscale#260]
Any chance to have this fixed in the next release?
```
resource "exoscale_security_group_rule" "lb1" {
  security_group_id      = exoscale_security_group.my_security_group.id
  type                   = "INGRESS"
  protocol               = "TCP"
  user_security_group = "public-nlb-healthcheck-sources"
  start_port             = 32080
  end_port               = 32080
}
```

**Comment by PhilippeChepy** [comment]
Hello @fegger-ducksify,

I'm currently working actively to fix this issue in both the Terraform provider and our CLI.

**Comment by PhilippeChepy** [comment]
This feature will be available in the next release.
We plan to cut a release this afternoon or at the beginning of next week.

**Comment by PhilippeChepy** [comment]
Fixed in v0.48.0 (available in the Hashicorp registry)
